triciaswift/tastevine-server
# Added destroy method to recipe view

I added a destroy method on Recipe View

Supported routes
`/recipes/n` for a DELETE request will return an empty body

## Steps to test
1. Pull down the `ts-destroy-recipe` branch and switch to it
2. If your debugger is running, restart it
3. Open Postman
4. Use this token in the header:
```
Token c29b168a0c1e461b3ff36dd9ce15e6e1d1768597
```
5. Perform a PUT to `http://localhost:8000/recipes/1`  with the following body:
6. Should receive a 204 status and receive an object back with the newly created information.
7. If you try a PUT to `http://localhost:8000/recipes/3`
8. You should receive a 403 forbidden message since the user is not the author of that recipe.
